racktables/find_pingable_machines.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+found_machines=0
+while read fqdn; do
+    ping -c1 -W1 "${fqdn}" &> /dev/null && echo "${fqdn} up" && found_machines=$(($found_machines + 1))
```

Comment:
You are in "math mode" here
```suggestion
    ping -c1 -W1 "${fqdn}" &> /dev/null && echo "${fqdn} up" && found_machines=$((found_machines + 1))
```